Makefile

```diff
@@ -218,10 +219,15 @@ ifeq (,$(shell grep -e $(VERSION) docs/source/conf.py))
 	$(error "Version specified in docs/source/conf.py does not match $(VERSION)")
 endif
 
-pypi-push-master: pypicheck pypi-version-check
+pypi-platform-check:
+ifeq (,$(PYPI_PLATFORM))
```

Comment:
maybe this should just have a default if its not provided?

Reply:
sure